app/assets/stylesheets/application/helpers.scss.erb

```diff
@@ -11,6 +11,20 @@
 %>
 @import "./_custom/<%=ENV['CLIENT']%>";
 <% end %>
+<%
+  # If we've defined further overrides in the theme
+  theme = Rails.cache.fetch('active_theme', expires_in: 1.minutes) do
+    GrdaWarehouse::Theme.first
```

Comment:
Should this be `find_by(client: ENV.fetch('CLIENT')`?

Reply:
I don't think so.  At least for now the `client` is only valid on the HMIS side.  (We only ever have one theme record for the warehouse and one ENV[CLIENT].  But we might be backing multiple HMIS.

Reply:
OK -- I'm just concerned that you use `first_or_create(client: ...)` below, so, it seems possible that they might not match.

Reply:
Good point